docs/contributing.md

```diff
@@ -57,7 +57,7 @@ Never made an open source contribution before? Wondering how contributions work
 Most of our style is pretty easy to pick up on, but right now it's not entirely consistent. You should match the style of the code surrounding your change, but if that code is inconsistent or unclear use the following guidelines:
 
 * We indent using two spaces (soft tabs)
-* We use One True Brace Style
+* We use [One True Brace Style](http://google.github.io/styleguide/javaguide.html) (java specific things should be ignored)
```

Comment:
With the example, I don't think this link is needed. Adding Java into the mix might just make things confusing.

Reply:
Perhaps we need the word modified here?

> * We use a modified One True Brace Style